docs_src/examples/Ch-ExamplesUsbDeviceService.md

```diff
@@ -0,0 +1,345 @@
+
+### Install additional Tools
+Install the media utility tool:
+
+   ```bash
+   sudo apt install mplayer v4l-utils
+   ```
+
+- `mplayer` is used to view the video stream
+- `v4l-utils` is used to determine the video stream path of a usb camera
+
+## Get the Source Code
+###  Download EdgeX Compose Repository (if not already downloaded)
+
+1. Create a directory for the EdgeX compose repository:
+   ```bash
+   mkdir ~/edgex
+   ```
+
+2. Change into newly created directory:
+   ```bash
+   cd ~/edgex
+   ```
+
+3. Clone the EdgeX compose repository
+   ```bash
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```
+
+
+### Get the Device USB Camera Source Code (if not already downloaded)
+
+1. Change into the edgex directory:
+   ```bash
+   cd ~/edgex
+   ```
+
+2. Clone the device-usb-camera repository:
+
+   ```bash
+   git clone https://github.com/edgexfoundry/device-usb-camera.git
+   ```
+## Deploy EdgeX and USB Device Camera Microservice
+### Building the docker image
+1. Change into newly created directory:
+   ```bash
+   cd ~/edgex/device-usb-camera
+   ```
+
+1. Build the docker image of the device-usb-camera service:
+   ```bash
+   make docker
+   ```
+1. Navigate to the Edgex compose directory.
+
+   ```shell
+   cd ~/edgex/edgex-compose/compose-builder
+   ```
+   
+1. Update `.env` file to add the registry and image version variable for device-usb-camera:
+
+   Add the following registry and version information:
+   ```env
+   DEVICE_USBCAM_VERSION=0.0.0-dev
+   ```
+
+1. Update the `add-device-usb-camera.yml` to point to the local image:
+
+   ```yml
+   services:
+   device-usb-camera:
+      image: edgexfoundry/device-usb-camera${ARCH}:${DEVICE_USBCAM_VERSION}
+   ```
+### Run the Service
+
+1. Navigate to the Edgex compose directory.
+
+   ```shell
+   cd ~/edgex/edgex-compose/compose-builder
+   ```
+
+2. Run EdgeX with the microservice:  
+> **NOTE:** This command runs the EdgeX microservices in non secure mode.
+   ```bash
+    make run no-secty ds-usb-camera 
+   ```
+
+## Verify Service and Device Profiles
+
+1. Check the status of the container:
+
+   ```bash 
+   docker ps
+   ```
+
+   The status column will indicate if the container is running and how long it has been up.
+
+   Example Output:
+
+   ```docker
+   CONTAINER ID   IMAGE                                         COMMAND                  CREATED       STATUS          PORTS                                                                                         NAMES
+   f0a1c646f324   edgexfoundry/device-usb-camera:0.0.0-dev                        "/docker-entrypoint.…"   26 hours ago   Up 20 hours   127.0.0.1:8554->8554/tcp, 127.0.0.1:59983->59983/tcp                         edgex-device-usb-camera                                                                   edgex-device-onvif-camera
+   ```
+
+2. Check that the device service is added to EdgeX:
```

Comment:
optionally we can have the user check the EdgeX-UI

Reply:
Or EdgeX CLI. The snap can be easily installed on the used system by running: `sudo snap install edgex-cli`.

Reply:
I had an issue getting the following command to work through the edgex-cli:

`edgex-cli command write -d Camera001 -c StartStreaming -b "{\"StartStreaming\":{\"InputImageSize\": \"640x480\",\"OutputVideoQuality\": \"5\"}}"`

replacing this curl command:

curl -X PUT -d '{
    "StartStreaming": {
      "InputImageSize": "640x480",
      "OutputVideoQuality": "5"
    }
}' http://localhost:59882/api/v2/device/name/<device name>/StartStreaming


I get the following error:
`Error: json: cannot unmarshal object into Go value of type string`

do you have any suggestions?

Reply:
is it ok if the guide is left how it currently is?

Reply:
I will check this later on but since your guide is using the HTTP endpoints consistently then it good to go.